doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
+Normalization of the integration reference points between different layers of the NFV architecture stack is one of the main concerns of the telco industry.  In the Networking context the primary focus is directed on the packet flow and control flow interfaces between Virtualisation Layer (be it IaaS or CaaS type of virtualisation) and Networking Hardware resources, as well as on related integration into the various MANO reference points (Hardware/ Network Infrastructure Management/Control, Orchestration). 
 
-The following subsections, Executive Summary, Strategy Objectives, Networking Requirements and Initial Approach, respectively, are intended to provide an overview of CNTT's vision for networking and intent, objectives, requirements and their supporting rationale, as well as initial development approach.
+It is an important feature of the proposed Networking Model to ensure the ability for supporting simultanous usage of different reference architectures by having multiple CaaS and IaaS layers on top of a shared Hardware Infrastructure layer. The details of this approach are provided in the section "Network Layering and Concepts" below.
 
-The networking within a cloud infrastructure, fabric or otherwise, is an area where there is significant variability across implementations. Leaf-Spine topology is well established, however, after topology there are countless decisions an Operator needs to make. Differences arise from many aspects, for example, is the solution layer-2 or layer-3; is the routing static or dynamic; what mechanism is used for encapsulation; what mechanism is used for isolation; does it support SR-IOV; does it support DPDK; does it employ SmartNICs; does it employ distributed control or a centralized control driving a programmable fabric; and the list continues. The multitude of permutations enable cloud infrastructure architects (Operators and Suppliers) to design (or procure) a fabric/networking solution that's optimized for their needs, whether their needs are minimal, very extensive or somewhere in between.
+<a name="5.1.1.1"></a>
+#### 5.1.1.1 Identified Networking Gaps
+To start the community discussion around potential gaps in the body of existing standards around the Cloud Infrastructure (NFV Infrastracture) networking, we looked into the ETSI NFV Architectural Framework. We have identified a number of areas that, in our view, need to be addressed (see below for the initial list). As usual, in our further efforts, the community will need to decide where and how these items should be addressed.
```

Comment:
@ulikleber, I can see that you are allergic to the word "gap".  In CNTT, from the beginning we set a goal to fill the gaps where identified, and this is not and should not be an offense for any other parties. I will think through how to rephrase this paragraph but eventually we cannot escape realization that such gaps exist (even if we call them "areas of concern").  Otherwise we would not need to create this task force at all.  Wouldn't be nice if everything had been done by someone else?

Reply:
@walterkozlowski The concern is the paragraph is editorial in nature. Suggest we simply format it as other editorial comments in the doc (indented italics in light gray w/ vertical bar on left), or revise w/ @ulikleber proposed text. To your point, I don't see any problem w/ the word "gap". It's just a matter of declaring that, that one paragraph is more of a comment, than part of the spec, itself.

Reply:
@walterkozlowski Looks good, Walter!